web-admin/tests/bookmarks.spec.ts

```diff
@@ -0,0 +1,261 @@
+import { expect, type Page } from "@playwright/test";
+import { assertUrlParams } from "@rilldata/web-common/tests/utils/assertUrlParams";
+import { test } from "./setup/base";
+import { interactWithTimeRangeMenu } from "@rilldata/web-common/tests/utils/exploreInteractions";
+
+test.describe("Bookmarks", () => {
+  // TODO: use a separate explore to isolate bookmarks
```

Comment:
Given you've edited the `explores.spec.ts` file, it looks like this comment is outdated?

Reply:
I want to add `_bookmark` prefix to the dashboard so that it is clear what it is for. Temporarily I just moved the `explores.spec.ts` to use the other dashboard.